test/redux/reducers/trendsStateByUser.test.js

```diff
@@ -42,6 +42,7 @@ describe('trendsStateByUser', () => {
         [USER_1]: {
           focusedCbgSlice: null,
           focusedCbgSliceKeys: null,
+          focusedSmbg: null,
```

Comment:
Having to add this line everywhere is really tedious I'm sure. This is one of the reasons I wasn't totally happy with the restructure of the reducer(s) for trends state that I did. Perhaps all of us (ping @krystophv) should brainstorm some ways to refactor these tests (with more constants?) or test smaller pieces of the state tree, IDK. Just food for thought. Not sure this approach is going to scale very well :/